### PR TITLE
Holiday bot

### PR DESCRIPTION
**Pull request by OlehEngineer**
create TelegramBot which return holidays in chosen country for today

### CODE REVIEW

File: Telegram_Bot/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.env
+*.exe
+*.log
```
Comment (Unlym) [inline Telegram_Bot/.gitignore:3]: why exist file LOG.log when you add that here?

File: Telegram_Bot/holidays/holidays.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	log "github.com/sirupsen/logrus"
+)
+
+func MakeHolidayRequest(userCountry string, token string) string {
+	country := userCountry[len(userCountry)-2:] //cut the country's abreviation from user's input
+	t := time.Now()
+	year := t.Year()
+	month := int(t.Month())
+	day := t.Day()
+	link := ("https://holidays.abstractapi.com/v1/?api_key=" + token + "&country=" + country + "&year=" + strconv.Itoa(year) + "&month=" + strconv.Itoa(month) + "&day=" + strconv.Itoa(day))
```
Comment (Unlym) [inline Telegram_Bot/holidays/holidays.go:19]: please use url.URL for build url

Comment (Unlym) [reply Telegram_Bot/holidays/holidays.go:19]: or fmt.Sprintf

File: Telegram_Bot/holidays/holidays.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+	resp, err := http.Get(link)
+	if err != nil {
+		log.Fatalf("Holiday request error => %s\n", err)
```
Comment (Unlym) [inline Telegram_Bot/holidays/holidays.go:23]: Don't use fatal. Please return string, error, and handle error upper

**Comment by OlehEngineer** [comment]
issues will be resolved.

**Pull request closed without merging**